src/GraphClient.ts

```diff
@@ -0,0 +1,75 @@
+import ApolloClient from 'apollo-client';
+import { WebSocketLink } from 'apollo-link-ws';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { SubscriptionClient } from 'subscriptions-transport-ws';
+import gql from 'graphql-tag';
+
+import WebSocket = require('ws');
+
+/**
+ * The class interacts with graph node server for subscription and query.
+ */
+export default class GraphClient {
+  private subgraphEndPoint: string;
+
+  /**
+   * GraphClient constructor.
+   *
+   * @param {string} subgraphEndPoint Subgraph http/ws end point.
+   */
+  public constructor(subgraphEndPoint: string) {
+    this.subgraphEndPoint = subgraphEndPoint;
+  }
+
+  /**
+   * Subscribes to the input subscription subgraph query and delegates the response
+   * to observer i.e. TransactionHandler.
+   * Documentation: https://www.apollographql.com/docs/react/advanced/subscriptions/
+   *
+   * @param subscriptionQry Subscription query object.
+   * @return `Subscription` Query subscription object
+   */
+  public subscribe(subscriptionQry: string) {
```

Comment:
Specify return type

Reply:
Subscription is the return type already specified. Am I missing something here 😕 ?

Reply:
Yes, its missing in the following line.
`public subscribe(subscriptionQry: string) {`